src/test/java/org/opentestsystem/ap/imrt/common/repository/ImrtItemRepositoryTest.java

```diff
@@ -119,4 +123,51 @@ public void shouldFindByStimId() {
         assertThat(result.size()).isEqualTo(2);
         assertThat(result).contains(item1, item3);
     }
+
+    @Test
+    public void shouldUpdateItemsWithStandardId() {
+        ImrtItem imrtItem = new ImrtItemBuilder()
+                .withKey(null)
+                .build();
+        // After a successful save, the returned object should be equal to the original object
+        imrtItemRepository.save(imrtItem);
```

Comment:
Did you mean to add a check on the returned object == original object?

Reply:
I think this is just part of the setup for this test, making sure we have an existing item that we want to update.  As such, I dunno that we need to verify what we saved matches the original (my assumption is that's tested somewhere else).

Reply:
That's fine, then just remove the comment about them being equal

Reply:
Will do.  Sorry about that.  I modified this test a few ways to make sure we tested all the different ways to save the item.